dist/b2b/organizations_members.js

```diff
@@ -179,6 +179,27 @@ class Members {
     });
   }
 
+  /**
+   * @param data {@link B2BOrganizationsMembersTOTPRequest}
+   * @param options {@link B2BOrganizationsMembersTOTPRequestOptions}
+   * @returns {@link B2BOrganizationsMembersTOTPResponse}
+   * @async
+   * @throws A {@link StytchError} on a non-2xx response from the Stytch API
+   * @throws A {@link RequestError} when the Stytch API cannot be reached
+   */
+  totp(data, options) {
```

Comment:
delete TOTP? Also documentation?

Reply:
Yeah this one is super strange. even the object names are missing the `DeleteMFA` portion that is present in the protos

Reply:
@taronish-stytch - I can work with you tomorrow to figure out why codegen is doing weird things.